install/wavelog-install.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: Don Locke (DonLocke)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://github.com/wavelog/wavelog
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
```

Comment:
If these are core dependencies that are needed in every LXC, wouldn't it make more sense to add them as part of the install.func instead of in each script?

Updated for now to match other scripts.

Reply:
Yes for later, we build some things in the background, for example this: https://github.com/community-scripts/ProxmoxVE/pull/249

Reply:
I think it's better to have all dependencies in each script, that makes it explicit what is installed in each LXC, instead of having "secret" dependencies that are installed in the background that only contributors know about.